docs/content/comparisons/druid-vs-storage-formats.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: doc_page
+---
+
+Druid vs Storage Formats (Parquet/Kudu)
+=======================================
+
+The biggest difference between Druid and existing storage formats is that Druid includes an execution engine that can run queries. 
+The Druid segment is a custom column format designed for fast aggregates and filters. Below we compare Druid's segment format to 
+other existing formats. 
+
+## Druid vs Parquet
+
+Druid's storage format is highly optimized for linear scans. Although Druid has support for nested data, Parquet's storage format is much 
+more hierachical, and is more designed for binary chunking. In theory, this should lead to faster scans in Druid.
+
+## Druid vs Kudu
```

Comment:
other things I would add:

- Kudu supports arbitrary primary keys with uniqueness constraints, and efficient lookup by ranges of those keys. For example, if your key is '(user_id, timestamp)' you can look up the data for that user in a millisecond or two, and do millions of QPS, which Druid isn't aimed for.
- Kudu chooses _not_ to include the execution engine, but supports sufficient operations so as to allow node-local processing from the execution engines. This means that we can support multiple frameworks on the same data (eg MR, Spark, and SQL).